subxt/src/utils/unchecked_extrinsic.rs

```diff
@@ -14,18 +14,48 @@ use std::marker::PhantomData;
 use codec::Decode;
 use scale_decode::{visitor::DecodeAsTypeResult, IntoVisitor, Visitor};
 
+use super::{Encoded, Static};
+
 /// The unchecked extrinsic from substrate.
 #[derive(Clone, Debug, Eq, PartialEq, Decode)]
 pub struct UncheckedExtrinsic<Address, Call, Signature, Extra>(
-    pub Vec<u8>,
-    #[codec(skip)] pub PhantomData<(Address, Call, Signature, Extra)>,
+    Static<Encoded>,
+    #[codec(skip)] PhantomData<(Address, Call, Signature, Extra)>,
 );
 
+impl<Address, Call, Signature, Extra> UncheckedExtrinsic<Address, Call, Signature, Extra> {
+    /// Construct a new [`UncheckedExtrinsic`].
+    pub fn new(bytes: Vec<u8>) -> Self {
+        Self(Static(Encoded(bytes)), PhantomData)
+    }
+
+    /// Get the bytes of the encoded extrinsic.
+    pub fn bytes(&self) -> &[u8] {
+        self.0 .0 .0.as_slice()
+    }
+}
+
+impl<Address, Call, Signature, Extra> From<Vec<u8>>
+    for UncheckedExtrinsic<Address, Call, Signature, Extra>
+{
+    fn from(bytes: Vec<u8>) -> Self {
+        UncheckedExtrinsic::new(bytes)
+    }
+}
+
+impl<Address, Call, Signature, Extra> From<UncheckedExtrinsic<Address, Call, Signature, Extra>>
+    for Vec<u8>
+{
+    fn from(bytes: UncheckedExtrinsic<Address, Call, Signature, Extra>) -> Self {
+        bytes.0 .0 .0
+    }
+}
+
 impl<Address, Call, Signature, Extra> codec::Encode
     for UncheckedExtrinsic<Address, Call, Signature, Extra>
 {
     fn encode_to<T: codec::Output + ?Sized>(&self, dest: &mut T) {
-        dest.write(&self.0)
+        dest.write(self.bytes())
```

Comment:
Or `self.0.encode_to(dest)` to fully defer to the internal impl (no space saved but one less place to check impl for in the future)